public/locales/en.json

```diff
@@ -1,8 +1,10 @@
 {
-	"404-page": {
-		"content_title": "Opps! We did not find this page",
-		"content_desc": "But you can return to the main page",
-		"link_back_to_home": "Back To Home"
+	"products": {
+		"add_to_cart": "Add To Cart",
```

Comment:
I think the translation “add to cart”, like in other translations, should be made with a small letter, or vice versa in German and Russian translations, with a capital letter.